*** tests/A-19-07-M-ok.out	2021-11-11 09:51:24.000000000 +0000
--- tests/A-19-07-M-ok.outhyp	2021-11-19 22:41:48.668090599 +0000
***************
*** 30,36 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M2|GARRAFA|5|125|125|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 30,36 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|0|M2|GARRAFA|5|109|109|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 38,44 ****
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|GARRAFA|1|39|39|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
--- 38,44 ----
  4 - Registar compra
  5 - Receber pagamento de venda
  0 - Sair
! Escolha uma opção: Identificador da transacção: VENDA|1|M2|GARRAFA|1|33|33|2
  Gestão de Transacções
  1 - Visualizar
  2 - Registar desagregação
***************
*** 63,72 ****
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|39|0|ROLHA:10#VIDRO:5
  ROLHA|2|35
  SAL|40|500
! VIDRO|3|5
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 63,72 ----
  3 - Visualizar lotes fornecidos por parceiro
  4 - Visualizar lotes de um dado produto
  0 - Sair
! Escolha uma opção: GARRAFA|33|0|ROLHA:10#VIDRO:5
  ROLHA|2|35
  SAL|40|500
! VIDRO|3|15
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
***************
*** 75,81 ****
  0 - Sair
  Escolha uma opção: ROLHA|M1|2|35
  SAL|S1|40|500
! VIDRO|M1|3|5
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
--- 75,82 ----
  0 - Sair
  Escolha uma opção: ROLHA|M1|2|35
  SAL|S1|40|500
! VIDRO|M1|2|0
! VIDRO|M1|3|15
  Gestão de Produtos
  1 - Visualizar todos os produtos
  2 - Visualizar todos os lotes
